src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBArgumentCollection.java

```diff
@@ -13,16 +13,25 @@
   public static final String USE_GCS_HDFS_CONNECTOR = "genomicsdb-use-gcs-hdfs-connector";
 
   public static final String CALL_GENOTYPES_LONG_NAME = "call-genotypes";
+  public static final String MAX_ALTS_LONG_NAME = "max-alternate-alleles-for-gdb";
   private static final boolean DEFAULT_CALL_GENOTYPES = false;
   private static final boolean DEFAULT_USE_BCF_CODEC = false;
   private static final boolean DEFAULT_SHARED_POSIXFS_OPTIMIZATIONS = false;
   private static final boolean DEFAULT_USE_GCS_HDFS_CONNECTOR = false;
 
   /**
-   * Not full-fledged arguments for now.
+   * Maximum number of alternate alleles that will report likelihoods after being combined on reading from GenomicsDB (including <NON_REF>)
+   * Must be at least one greater than the maxmimum number of alternate alleles for genotyping.  Note that GenotypeGVCFs will drop
+   * highly multi-allelic sites that are missing likelihoods.
```

Comment:
Can you include a bit more guidance here on selecting sane values for this argument? Eg.,

```
I would suggest setting the --max-alternate-alleles-for-gdb to a value 3-4 more than the 
--max-alternate-alleles value that's used by GenotypeGVCFs if you want to be sensitive about 
retaining variants, keeping in mind that sites with more than --max-alternate-alleles-for-gdb will 
ultimately be dropped. For a human cohort, I would typically expect sites with more than 6 alleles 
to be (indel) PCR stutter errors, but use your own judgement based on your organism's mutation 
rate and population diversity.
```